coalib/bears/Bear.py

```diff
@@ -273,6 +276,10 @@ def __init__(self,
                               time limit, use 0.
         :raises TypeError:    Raised when ``message_queue`` is no queue.
         :raises RuntimeError: Raised when bear requirements are not fulfilled.
+        :param profile        A flag which enable profiler on bears of a
+                              section. Contains path or name of directory where
+                              profiled files will dump if none provided then
+                              profiled files will dump in current directory.
```

Comment:
1. --> A flag which enable**s** profile**ing** on **this bear**.
2. It's not really a flag then, is it? Since it accepts paths as well.
3. Nothing is mentioned that `True` is a valid value (a bit confusing is that you mention "if none provided then...", which suggests that `None` will do the trick)